src/server/stream_family.cc

```diff
@@ -1272,6 +1272,38 @@ OpResult<uint32_t> OpDel(const OpArgs& op_args, string_view key, absl::Span<stre
   return deleted;
 }
 
+// XACK key groupname id [id ...]
+OpResult<uint32_t> OpAck(const OpArgs& op_args, string_view key, string_view gname,
+                         absl::Span<streamID> ids) {
+  OpResult<pair<stream*, streamCG*>> res = FindGroup(op_args, key, gname);
+  if (!res)
+    return res.status();
+  stream* stream_inst = res->first;
+  streamCG* cg = res->second;
+  if (cg == nullptr || stream_inst == nullptr) {
+    return 0;
+  }
+
+  int acknowledged = 0;
+  for (size_t j = 0; j < ids.size(); j++) {
```

Comment:
Nit: since you only use the values here, you could do:

```cpp
for (const auto& id : ids) {
  ...
}
```

Reply:
ok.